docs/existing-css.md

```diff
@@ -0,0 +1,73 @@
+# Using `styled-components` with existing CSS
+
+`styled-components` generates an actual stylesheet with classes, and attaches those classes to the DOM nodes of styled components via the `className` prop. It injects the generated stylesheet at the end of the HEAD of the document.
+
+## Styling normal React components
+
+If you use the `styled(MyNormalComponent)` notation with an existing, non-styled-components react component that doesn't use the passed-in `className` prop the styles won't be applied.
+
+To avoid this issue, make sure your component (in this case `MyNormalComponent`) attaches the passed-in `className` to a DOM node:
+
+```JSX
+class MyNormalComponent extends React.Component {
+  render() {
+    return (
+      // Attach the passed-in className to the DOM node
+      <div className={this.props.className}></div>
+    );
+  }
+}
+```
+
+If you have pre-existing styles with a class, you can combine the global class with the `styled-components` one:
+
+```JSX
+class MyNormalComponent extends React.Component {
+  render() {
+    return (
+      // Use a global class with existing styling in combination with
+      // allowing this component to be styled with styled-components
+      <div className={`some-global-class ${this.props.className}`}></div>
+    );
+  }
+}
+```
+
+## Global class always overridden
+
+The other way around might happen too. You have a styled component, and a standard, global CSS class you're trying to override some styles with. For some reason though, the `styled-components` styles always end up overriding the styles of the global class, no matter what you do!
```

Comment:
You can apply a global CSS class to a styled component by adding a `className` prop. This will work, but if a specific CSS property (e.g. `background-color`) is defined in both, the global CSS and the styled component, the results may not be what your expecting!

^ I think the point to get across is that you can use global css, but it has caveats